Core/Src/udp_service.c

```
@@ -63,67 +63,67 @@ static void customServiceReceptionHandler(EMBENET_UDP_SocketDescriptor const *so
+    udpServiceSocket = (EMBENET_UDP_SocketDescriptor ) {
         .port = 1234,
         .groupId = 0, // GroupId is ignored, when using EMBENET_UDP_TRAFFIC_UNICAST
         .handledTraffic = EMBENET_UDP_TRAFFIC_UNICAST,
-        .rxDataHandler = customServiceReceptionHandler,
+        .rxDataHandler = udpServiceReceptionHandler,
         .userContext = NULL // userContext is not needed in this example, however user may pass it to callback invocation
     };
 
     // Register UDP socket. Registering socket enables datagram reception/transmission
-    EMBENET_Result customServiceSocketRegistrationStatus = EMBENET_UDP_RegisterSocket(&customServiceSocket);
-    if (EMBENET_RESULT_OK == customServiceSocketRegistrationStatus) {
-        printf("CUSTOM_SERVICE: Socket %d registered successfully\n", (int)customServiceSocket.port);
+    EMBENET_Result udpServiceSocketRegistrationStatus = EMBENET_UDP_RegisterSocket(&udpServiceSocket);
+    if (EMBENET_RESULT_OK == udpServiceSocketRegistrationStatus) {
+        printf("UDP_SERVICE: Socket %d registered successfully\n", (int)udpServiceSocket.port);
         // Create a task using built-in scheduler
-        customServiceTaskId = EMBENET_NODE_TaskCreate(customServiceTask, NULL);
-        if (EMBENET_TASKID_INVALID == customServiceTaskId) {
-            printf("CUSTOM_SERVICE: Unable to create task\n");
+        udpServiceTaskId = EMBENET_NODE_TaskCreate(udpServiceTask, NULL);
+        if (EMBENET_TASKID_INVALID == udpServiceTaskId) {
+            printf("UDP_SERVICE: Unable to create task\n");
         } else {
-            printf("CUSTOM_SERVICE: Service initialized\n");
+            printf("UDP_SERVICE: Service initialized\n");
         }
     } else {
-        printf("CUSTOM_SERVICE: Registering socket failed with status %d\n", (int)customServiceSocketRegistrationStatus);
+        printf("UDP_SERVICE: Registering socket failed with status %d\n", (int)udpServiceSocketRegistrationStatus);
     }
```

you could reduce the amount of indentations by failing fast and returning